systemd/emulation.go

```diff
@@ -120,7 +120,7 @@ func (s *emulation) IsActive(service string) (bool, error) {
 	return false, &notImplementedError{"IsActive"}
 }
 
-func (s *emulation) LogReader(services []string, n int, follow bool) (io.ReadCloser, error) {
+func (s *emulation) LogReader(services []string, n int, follow, namespaces bool) (io.ReadCloser, error) {
```

Comment:
(nitpick) I'm not a fan of boolean options in APIs like we use them here. On the caller side this will look like:
```
x.LogReader([]{"srv1"}, true, false)
```
i.e. not easy at all to see what true/false means without looking at the function signature (and breaking the flow of reading the code). I prefer
```
x.LogReader([]{"srv1"}, &LogReaderOptions{follow: true, namespace: false})
```
but feel free to just ignore for now as most of the time the options will come from a var here so it's fairly clear.